app/models/post.rb

```diff
@@ -1,3 +1,13 @@
 class Post < ApplicationRecord
   belongs_to :user
+
```

Comment:
- Let's add also relationship with comments and likes
  one post will have many comments and many likes.